lib/index.js

```diff
@@ -46,7 +46,8 @@ module.exports = function (stream, opts, callback) {
     genre: [],
     disk: { no: 0, of: 0 },
     picture: [],
-    duration: 0
+    duration: 0,
+    lyrics: ''
```

Comment:
Can we remove this line? I only want to add to the default metadata output if it can be provided across all formats (id3,vorbis, asf) etc.